src/looprestoration.rs

```diff
@@ -136,3 +137,160 @@ decl_looprestorationfilter_fns! {
     fn dav1d_wiener_filter7_16bpc_neon,
     fn dav1d_wiener_filter5_16bpc_neon,
 }
+
+// 256 * 1.5 + 3 + 3 = 390
+const REST_UNIT_STRIDE: usize = 390;
+
+// TODO Reuse p when no padding is needed (add and remove lpf pixels in p)
+// TODO Chroma only requires 2 rows of padding.
+// TODO(randomPoison): Temporarily pub until remaining looprestoration fns have
+// been deduplicated.
+#[inline(never)]
+pub(crate) unsafe fn padding<BD: BitDepth>(
+    mut dst: &mut [BD::Pixel; 70 /*(64 + 3 + 3)*/ * REST_UNIT_STRIDE],
+    mut p: *const BD::Pixel,
+    stride: ptrdiff_t,
+    mut left: *const [BD::Pixel; 4],
+    mut lpf: *const BD::Pixel,
+    mut unit_w: libc::c_int,
+    stripe_h: libc::c_int,
```

Comment:
Can you make `stride`, `unit_w`, `stripe_h` `usize`s?  I've been doing that, too.  It makes the code a lot cleaner vs. casts at use site, and these values aren't negative.